### PR TITLE
Make bootstrap.py handle child zombification gracefully.

### PR DESCRIPTION
**Pull request by rmmh**

### CODE REVIEW

File: jenkins/bootstrap.py [hunk 1]
```diff
@@ -88,6 +88,9 @@ def call(cmd, stdin=None, check=True, output=None):
                     reads.remove(fdesc)
                     continue
                 logging.warning(line[:-1])
+        if not ret[0]:
+            if proc.poll() is not None:
+                break  # process exited without closing pipes (timeout?)
```
Comment (fejta) [inline jenkins/bootstrap.py]: Sot he third arg in select.select() is for exceptional circumstances... does this include not closing the pipes?

**Comment by fejta** [comment]
Thanks!
/lgtm

**Comment by fejta** [comment]
ref #1313 right?

File: jenkins/bootstrap.py (hunk 1 above)
Comment (rmmh) [inline jenkins/bootstrap.py]: I didn't see a way to use the xlist for anything useful-- it *might* be able to catch when a pipe is closed, but that's not what's happening here.

**Comment by rmmh** [comment]
@k8s-bot go test this

**Pull request merged**